Sources/FluentKit/Properties/Property.swift

```diff
@@ -32,14 +32,44 @@ public protocol AnyCodableProperty: AnyProperty {
 
 public protocol AnyQueryableProperty: AnyProperty {
     var path: [FieldKey] { get }
+    func queryableValue() -> DatabaseQuery.Value?
```

Comment:
Is this technically a breaking change for anyone who creates their own properties?

Reply:
No, because a default implementation is provided (and it's not legal to conform to `AnyQueryableProperty` without conforming to `QueryableProperty`).